bindata/network/multus/004-configmap.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cni-copy-bash-script
```

Comment:
In general we just inline shell scripts into the pod specs. eg, look at [`bindata/network/openshift-sdn/sdn.yaml`](https://github.com/dougbtv/cluster-network-operator/blob/copy-cni-plugins-rhel7-rhel8/bindata/network/openshift-sdn/sdn.yaml#L56)

Reply:
Thanks for the example -- nice to not have it floating out in another file, got this updated to have it inline'd per the example.